### DISCUSSION TITLE
Seeing high DB CPU usage in running job runr on Azure Sql storage

### DISCUSSION THREAD

**Discussion by a0p029x** [jobrunr/jobrunr#736] [General]
Hi All,

We're using Jobrunr version 5.2.0 with Azure sql . We're live on prod.
On first of day of live we saw close 348K scheduled jobs were processed succesfully, but we're seeing DB CPU shotting upto 100% many times in the day.
On further analysis on which query is consuming lot of CPU, we found below query

(@P1 int,@P0 bigint)select * from jobrunr_jobs_stats ORDER BY total ASC OFFSET @P0 ROWS FETCH NEXT @P1 ROWS ONLY

being called multiple times and using almost 80%+ of CPU memory. This is the view and we thought it might be used from Job runr dashboard. So we disabled dashboard to see if it helps, but we're still seeing high CPU usage.

Azure Sql instance details
Storage: 1TB
Standard-series (Gen5)
up to 128 vCores, up to 625 GB memory

Please help us on this as we're planning to onboard more jobs in coming days and DB choking is not helping us.

**Comment by rdehuyss** [comment]
Feel free to reach out to me via ronald@jobrunr.io.

**Comment by rdehuyss** [comment]
@a0p029x : the solution is easy but I'm afraid I cannot help everybody for free. If you have that size of an SQL server, I can really help you with downsizing the server which will save you a lot of Azure Cloud costs. But, support is only available if you take a Pro subscription.
